Sources/XcbeautifyLib/Parser.swift

```diff
@@ -1,156 +1,226 @@
+public enum OutputType {
```

Comment:
Could you extract this enum into a new file?

Reply:
I notice there is a Constants.swift, do you think it's a good place to put this enum in?

Reply:
Since we don’t have that many types, I think it’s better to give each type its own file. I plan to extract Constants.swift out later.